src/input_parsers/NetworkParser.h

```diff
@@ -0,0 +1,70 @@
+/*********************                                                        */
+/*! \file NetworkParser.cpp
+ ** \verbatim
+ ** Top contributors (to current version):
+ **   Matthew Daggitt
+ ** This file is part of the Marabou project.
+ ** Copyright (c) 2017-2019 by the authors listed in the file AUTHORS
+ ** in the top-level source directory) and their institutional affiliations.
+ ** All rights reserved. See the file COPYING in the top-level source
+ ** directory for licensing information.\endverbatim
+ **
+ ** This file provides a general interface for parsing a neural network.
```

Comment:
Can you elaborate on the class description here? What is the purpose of the NetworkParser and what format is it parsing?